azuread/helpers/graph/group.go

```diff
@@ -0,0 +1,64 @@
+package graph
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/graphrbac/1.6/graphrbac"
+)
+
+func GroupAllMembers(groupId string, client graphrbac.GroupsClient, ctx context.Context) ([]string, error) {
```

Comment:
Could we change the signiture to:
```suggestion
func GroupAllMembers(client graphrbac.GroupsClient, ctx context.Context, groupId string) ([]string, error) {
```

As it will better match the actual API call